tests/middleware/test_logging.py

```diff
@@ -45,6 +46,54 @@ async def test_trace_logging(caplog):
         assert "ASGI [2] Completed" in messages.pop(0)
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize("protocol", [("h11"), ("httptools")])
```

Comment:
```suggestion
@pytest.mark.parametrize("http_protocol", [("h11"), ("httptools")])
```
small naming variable fight :)